src/components/task-plan/homework/add-exercises.cjsx

```diff
@@ -81,8 +82,10 @@ AddExercises = React.createClass
     TaskPlanStore.hasExercise(@props.planId, exercise.id)
 
   setCurrentSection: (currentSection) ->
+    @scrollToSelector("[data-section='#{currentSection}']")
```

Comment:
Hmm, I wonder if this dependency could/should be documented in the Component that generates the `data-section` attribute; so we don't forget that there's this dependency